test/services/organizations/create_service_test.rb

```diff
@@ -21,7 +20,21 @@ class Organizations::CreateServiceTest < ActiveSupport::TestCase
       }
     }
 
-    Organizations::CreateService.new.signal(args)
-    assert_not Organization.last.custom_page.nil?
+    Organizations::CreateService.any_instance.stubs(:send_email).returns(true)
+
+    assert_difference "Organization.count", 1 do
+      Organizations::CreateService.new.signal(args)
+
+      organization = Organization.last
+
+      location = organization.locations.last
+      assert_not_nil location, "Location was not created"
+      assert_equal "Mexico", location.country
+      assert_equal "La Ventana", location.city_town
+      assert_equal "Baja", location.province_state
+      
```

Comment:
@Isaac3924 We use [standard](https://github.com/standardrb/standard) for linting. The command for auto-fixing is `rails standard:fix` . When I ran it on your branch it removed the spaces on line 35.
```suggestion

```

Reply:
Dunno why but standard wasn't working on my build last I was trying to fix it... it is now though, so this SHOULD be resolved on my next push, but currently dealing with another set of errors.